src/components/doughnutChart/DoughtnutChart.tsx

```diff
@@ -0,0 +1,11 @@
+import { Doughnut } from 'react-chartjs-2';
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js';
+import { chartDataFormatter } from '../../hooks/ChartDataFormatter';
+import { IChartProps } from '../../interfaces';
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+// eslint-disable-next-line max-len
+export const DoughtnutChart = ({ dataSet }: IChartProps) => (
+  <Doughnut data-testid="Chart" data={chartDataFormatter({ dataSet })} />
```

Comment:
` => <Doughnut data-testid="Chart" data={chartDataFormatter({ dataSet })} /> ` use return

Reply:
So I should go back on hwo it was previously before your last review?